presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaTableHandle.java

```diff
@@ -31,19 +31,19 @@
     private final String catalogName;
     private final String schemaName;
     private final String tableName;
-    private final Set<QualifiedTablePrefix> prefixes;
+    private final Optional<TupleDomain<ColumnHandle>> constraint;
```

Comment:
We don't need `Optional` here, as `TupleDomain.all()` can be used to expressed "no constraint" case.